tubesize/src/pages/options.tsx

```
@@ -0,0 +1,54 @@
+                </div>
+            </div>
+
+            <div className="section-divider"></div>
+            <CacheSettings />
+
+            <div className="section-divider"></div>
+            <ApiFallbackSetting />
+
+            <div className="section-divider"></div>
+            <div className="author">
+                <a href="https://github.com/MohamedSayed0573" target="_blank">
```

Biome (2.4.7)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

(lint/security/noBlankTarget)